exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -238,17 +238,22 @@ func logToCWLog(resourceAttrs map[string]interface{}, log plog.LogRecord, config
 		var metadata emfMetadata
 		bodyString := log.Body().AsString()
 		err = json.Unmarshal([]byte(bodyString), &metadata)
+		switch {
 		// v1 emf json
-		if err == nil && metadata.AWSMetadata != nil && metadata.AWSMetadata.LogGroupName != "" {
+		case err == nil && metadata.AWSMetadata != nil && metadata.AWSMetadata.LogGroupName != "":
 			logGroupName = metadata.AWSMetadata.LogGroupName
 			if metadata.AWSMetadata.LogStreamName != "" {
 				logStreamName = metadata.AWSMetadata.LogStreamName
 			}
-		} else /* v0 emf json */ if err == nil && metadata.LogGroupName != "" {
+		// v0 emf json
+		case err == nil && metadata.LogGroupName != "":
 			logGroupName = metadata.LogGroupName
 			if metadata.LogStreamName != "" {
 				logStreamName = metadata.LogStreamName
 			}
+		case config.EmfOnly:
+			logger.Debug("not a valid emf", zap.String("log", bodyString))
```

Comment:
We don't need this logger. It already logs the returned error in `logsToCWLogs`.

```
logger.Debug("Failed to convert to CloudWatch Log", zap.Error(err))
```

Could add the `zap.String("log", bodyString)` there. As is, you'd have two debug logs for the same error.